src/components/state-details.js

```diff
@@ -271,17 +277,25 @@ export default function StateDetailsPage ({ location, data }) {
                   Mostly from different types of heating.
                 </p>
 
+                <p className="mt-5 mb-0">
+                  <img className="img-fluid" src={GasAppliances} title="Gas furnace, gas water heater, gas stove" alt="Gas furnace, gas water heater, gas stove" />
```

Comment:
Huh, why have a `title` here? Do we really need this as a tooltip?

Reply:
good point. i have a habit of including both, but you're right its not necessary to have a tooltip for this